README.md

```diff
@@ -496,64 +487,87 @@ The `auth` field MUST be contain an [Authorization] which signs over the `&Task`
 
 Concretely, this means that the `&Task` MUST be present in the associated `auth`'s `scope` field. An `Invocation` where the associated [Authorization] does not include the [Task] in the `scope` MUST be considered invalid.
 
-## 5.1 Schema
+## 5.1 Task
 
-```ipldsch
-type Invocation struct {
-  v       SemVer
+```Task
+type Context struct {
```

Comment:
@expede should be `type Task` no? Task is in the markdown, but that's odd. Later we have https://github.com/ucan-wg/invocation/pull/15/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R207 which is the same?